core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -237,7 +237,7 @@ object DumpLogSegments {
       if (bytes.isEmpty)
         ""
       else
-        String.format("%X", BigInt(1, bytes))
+        String.format("%X", BigInt(1, bytes).bigInteger)
```

Comment:
Doesn't this just work I'd you use format on the (Scala) String instead of the static method version?